contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/BrowsingCard.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+
+  <VCard>
+    <VCardTitle>
+      <VLayout row>
+        <VFlex sm2 xs12>
+          <VLayout align-center justify-center fill-height>
+            <VImg
+              v-if="node.thumbnail"
+              height="80px"
+              maxHeight="80px"
+              width="80px"
+              maxWidth="80px"
```

Comment:
lots of extra for a fixed size image... but I can't see any other way of doing it :man_shrugging:

Reply:
i actually tried to make this smaller relative to the mockup and wanted to wait till i had more nodes with real images to try it out. what size do you think is better?